tests/many_to_one/tests.py

```diff
@@ -669,3 +669,10 @@ def test_add_then_remove_after_prefetch(self):
         self.assertEqual(city.districts.count(), 2)
         city.districts.remove(d2)
         self.assertEqual(city.districts.count(), 1)
+
+    def test_cannot_reuse_trimmed_joins(self):
+        """ See #26551 for more details """
```

Comment:
Try to describe the issue here rather than reader to look up the ticket to understand the issue.